src/System.Management/src/Configurations.props

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netstandard-Windows_NT;
```

Comment:
>netstandard-Windows_NT [](start = 6, length = 22)

Small question, have we tried this in uap and in desktop? If not, perhaps we only want to have it on netcoreapp-Windows_NT;

Reply:
Yes, I am changing to that.

Reply:
I think we want `netcoreapp2.0-Windows_NT` since it must run specifically on 2.0 
And tests configurations.props should change to the same.

Reply:
Yes, I am working on that but hit an early snag with netfx needing System.Management. Jose gave me the info and I am trying to add what is needed.

Reply:
I was just chatting with @safern about the configurations for these type of libraries and I think what we want is:

```
  <PackageConfigurations>
    netstandard;
    netcoreapp2.0-Windows_NT
  </PackageConfigurations>
  <BuildConfigurations>
   $(PackageConfigurations);
   netcoreapp;
  </BuildConfigurations>
```

The netstandard20 will be PNSE assembly and netcoreapp20 will have a windows only implementation. Then the netcoreapp (aka netcoreapp2.1) is there for the netcoreapp vertical build (and build from source) to enable us to build without older versions of netcoreapp restore. 

We will also need the place holder for netfx in the package and @safern is going to work on a way to annotate that in these configurations files as well so we can correctly test on netfx.

Reply:
Yup. I will update my PRs and will submit a PR to update this project to have the right set of configurations.

Reply:
@safern please cc me on these upcoming PRs.

Reply:
> @safern please cc me on these upcoming PRs.

Sure I will do.